packages/core/src/source/at.ts

```diff
@@ -0,0 +1,26 @@
+/** @license MIT License (c) copyright 2010-2017 original author or authors */
+
+import { propagateTask } from '../scheduler/PropagateTask'
+import { delay } from '@most/scheduler'
+import { Time, Stream, Sink, Scheduler, Disposable } from '@most/types'
+
+export const at = <A>(t: Time, x: A): Stream<A> => new At(t, x)
+
+class At<A> implements Stream<A> {
+  private readonly time: Time;
+  private readonly value: A;
+
+  constructor (t: Time, x: A) {
+    this.time = t
+    this.value = x
+  }
+
+  run (sink: Sink<A>, scheduler: Scheduler): Disposable {
+    return delay(this.time, propagateTask(runAt, this.value, sink), scheduler)
+  }
+}
+
+function runAt <A> (t: Time, x: A, sink: Sink<A>): void {
```

Comment:
Personal taste, I think the space between `<A>` and `(t: Time ...)` should be eliminated. That would also show some consistency with how it’s done in the `at` arrow function above.

Reply:
Agreed.  Honestly, I'm ok with either having a space or not having one, as long as we pick one and do it consistently.  Can we enforce (and autofix) with linting?

Reply:
Looks like there's a bug https://github.com/typescript-eslint/typescript-eslint/issues/506

Reply:
Bummer about the linting bug.  How about we go with no space by convention for now?  Sound good?

Reply:
I'm ok with any way which can be automated :)

Reply:
done